docs/_interface/consensus-clients.md

```diff
@@ -0,0 +1,149 @@
+of the Beacon client or download blocks and consensus data by sending requests using tools such as [Curl](https://curl.se).
+More information on this can be found in the documentation for each consensus client.
+
+## Validators
+
+After The Merge, miners are n longer responsible for securing the Ethereum blockchain. Instead, this becomes the responsibility
+of validators that have staked at least 32 ETH into a deposit contract and run validator software. Each of the consensus clients
+have their own validator software that is described in detail on their respective documentation. The easiest way to handle 
+staking and validator key generation is to use the Ethereum Foundation [Staking Launchpad][launchpad-link].
+
+## Using Geth
+
+After the merge, Geth will follow the head of the chain via its connection to the consensus client. However, Geth is still 
+the portal for users to send transactions to Ethereum. Overall, Geth will not change very much from a user-perspective. 
+The Geth Javascript console is still available for this purpose, and the majority of the [JSON-RPC-API](/docs/rpc/server) will 
+remain available via web3js or HTTP requests with commands as json payloads. These options are explained in more detail on the 
+[Javascript Console page](/docs/interface/javascript-console). The Javascript console can be started using the following command
+in a separate terminal (assuming Geth's IPC file is saved in `datadir`):
+
+```shell
+geth attach datadir/geth.ipc
+```
+
+
+## Testnets
+
+Ethereum Mainnet has not yet undergone The Merge, but some public testnets have. This means that running Geth alone is no longer
+enough to interact with merged testnets. This includes two testnets that were purpose built to test The Merge (Kiln, Kintsugi) and 
+the long-standing public PoW chain, Ropsten. If Geth is connected to these merged networks alone it will simply stall when it syncs as far
+as the merge block, awaiting information from a consensus client. Therefore, any activity on these testnets requires Geth to be 
+connected to a consensus client. There are many instructional articles that exlain how to connect to these testnets using Geth in
+combination with various consensus clients, for example:
+
+[Connecting to Kiln using Geku](https://github.com/chrishobcroft/TestingTheMerge/blob/main/geku.md)
+ 
+[Connecting to Kiln using GethHouse](https://github.com/remyroy/ethstaker/blob/main/merge-devnet.md)
+ 
+[Connecting to Kiln using Geth-Prysm](https://hackmd.io/@prysmaticlabs/B1Q2SluWq)
+ 
+[Connecting to Ropsten using GethStar](https://mirror.xyz/jmcook.eth/1GcUHyHst4VxXsbr4Ee3DJbrFQjAmQLkqt-fIBEeX_M)
+ 
+[Connecting to Ropsten using GethHouse](https://github.com/remyroy/ethstaker/blob/main/merge-ropsten.md)
+
+*(note that the client combinations are often named using portmanteaus of the individual client names, e.g. Geth + lighthouse = Geth-House, 
```

Comment:
Are these naming schemes commonly-used?  I'm not seeing many references to these names from cursory searches of discord and Github.

Maybe we can just use "Connecting to `testnet name` using `CL client name`" for the format of these subtitles.

Reply:
ok - I have certainly seen all of these used in various places, but maybe it's better just to omit. Thanks!